src/GHCJS/Fetch/FFI.hs

```diff
@@ -14,6 +16,7 @@ import GHC.Stack
 import GHCJS.Fetch.Types
 import GHCJS.Types
 import JavaScript.Object
+import JavaScript.TypedArray.ArrayBuffer (ArrayBuffer (..),MutableArrayBuffer (..))
```

Comment:
This import is currently causing a build failure since it is apparently not part of `ghcjs-base-stub`. Might be sufficient to make this import conditional and then declare a dummy type with no constructors `data MutableArrayBuffer` for the type declaration of `js_responseMutableArrayBuffer`.